src/madrone.js

```diff
@@ -66,6 +62,63 @@ window.addEventListener('load', () => {
       }
     }
   });
+
+  // ToCJS Width setter.
+  const tocJsBlocks = document.querySelectorAll(".toc-js");
+  if (tocJsBlocks.length > 0) {
+    let tocJsBlocks1 = tocJsBlocks[0];
```

Comment:
I'd pick a better name for this variable `tocJsBlock`. Having a '1' implies to me there would be another one at some point but I think you're just doing this because the query selector returns a list when you only care about the 1 match.

Reply:
good idea.
changes pushed.